frontend/public/components/operator-lifecycle-manager/operator-group.tsx

```diff
@@ -46,9 +48,68 @@ export const requireOperatorGroup = <P extends RequireOperatorGroupProps>(Compon
   } as React.ComponentClass<P> & {WrappedComponent: React.ComponentType<P>};
 };
 
+export enum InstallModeType {
+  InstallModeTypeOwnNamespace = 'OwnNamespace',
+  InstallModeTypeSingleNamespace = 'SingleNamespace',
+  InstallModeTypeMultiNamespace = 'MultiNamespace',
+  InstallModeTypeAllNamespaces = 'AllNamespaces',
+}
+
+export type InstallModeSet = {type: InstallModeType, supported: boolean}[];
+
+export const supports = (set: InstallModeSet) => (obj: OperatorGroupKind) => {
```

Comment:
Logic taken from [the Operator Lifecycle Manager](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/pkg/api/apis/operators/v1alpha1/clusterserviceversion.go#L99).